jobsdb/jobsdb.go

```diff
@@ -1361,9 +1361,6 @@ func (jd *HandleT) markClearEmptyResult(ds dataSetT, stateFilters []string, cust
 }
 
 func (jd *HandleT) isEmptyResult(ds dataSetT, stateFilters []string, customValFilters []string, parameterFilters []ParameterFilterT) bool {
-	queryStat := stats.NewJobsDBStat("isEmptyCheck", stats.TimerType, jd.tablePrefix)
-	queryStat.Start()
-	defer queryStat.End()
```

Comment:
This was added by me when I didn't realize it was just an in-memory operation. These stats are not necessary.